### DISCUSSION TITLE
Setting custom DLSS quality above 1.0 (native)

### DISCUSSION THREAD

**Discussion by Frub3L** [emoose/DLSSTweaks#148] [Q&A]
Hi guys,
I was having fun with the program and noticed that it's possible to set the custom scaling for dlss and set is >1.0, so more than native resolution. Could someone explain what exactly will happen when I do that? Is it something like DLDSR, or it won't work at all?

**Comment by qqaazz112** [comment]
Came here to ask the same thing after using Puredark's GTA 5 DLSS mod. The mod allows you to set res scaling to 2.0, and then downscales to your output res without needing DSR. The DLSS indicator shows 1440p>2880p>1440p. I don't think it works when I set scaling to 2.0 with DLSS Tweaks, I wonder if this can be added.

**Comment by pacopepepipo** [comment]
Hi

I just came looking for the same, as I don't know if it works

I just bought a new monitor, but it forces the user to have DSC enabled, disabling DLDSR on the nvidia settings

do anyone knows if enabling scale 1.5 will downsample from a higher resolution or will just ignore it and use 1.0  as dlaa? or just use the default?

how does it works with dlsstweaks?

**Comment by TatsunaKyo** [comment]
It would be great if it's possible. We should ask how PureDark managed to do it and if it requires work on a game basis.
Regarding arbitrary resolution targets, unfortunately it crashes in the only instance I have tried -- Final Fantasy Xvi, with a blank window error.
